shimmingtoolbox/cli/b0shim.py

```diff
@@ -249,7 +249,7 @@ def dynamic(fname_fmap, fname_anat, fname_mask_anat, method, opt_criteria, slice
 
     # Load the coils
     list_coils = _load_coils(coils, scanner_coil_order, fname_sph_constr, nii_fmap, options['scanner_shim'],
-                             json_fm_data['Manufacturer'], json_fm_data['ManufacturersModelName'])
+                             json_fm_data.get('Manufacturer'), json_fm_data.get('ManufacturersModelName'))
```

Comment:
what does `.get()` bring with a dictionary?

Reply:
It returns the same thing as `json_fm_data['Manufacturer']` but returns `None` rather than raising a `KeyError` if the key is not part of the dictionary